src/main/resources/templates/eventDetails.html

```diff
@@ -60,7 +59,9 @@ <h1 th:text="${event.name}">name</h1>
 
             </div>
         </div>
+
         <div class="col-md-6">
+            <p sec:authorize="hasAuthority('ADMIN')" th:insert="deleteEvent :: deleteEvent"></p>
```

Comment:
This will create a small unstyled link above the image to delete the event. its probably not what you intended to do. You most likely meant to just add `sec:authorize="hasAuthority('ADMIN')"` on the div at rule 53.